src/Tizen.NUI.Wearable/src/public/CircularPagination.cs

```diff
@@ -48,7 +48,7 @@ public class CircularPagination : Control
         private bool isOddNumber = true;
         private bool uninitializedLeftIndicator = true; // Need it when the indicators are asymmetry and the right indicator count is set earlier than left one.
         private Animation selectAnimation = null;
-        private bool isNeedAnimation = false; // TODO : Animation will support using override function later.
+        //private bool isNeedAnimation = false; // TODO : Animation will support using override function later.
```

Comment:
It's not used anywhere, then can we remove it at all?

Reply:
I'm not sure what TODO comment writer want to do in future. So I just want to keep this comment. That's why I dont remove this line. What do you think about it? Could I remove this line?

Reply:
There is also an unused API in this file. If we can remove unused values, Ill also try to clean up soon

Reply:
From what I've remembered, Circular Pagination had a requirement to select whether animation is working or not.
That's why this variable was added.
However, `NUI.Wearable` is meaningless now :/ and we can add the unused variables later if this class will be used someday.

So, Please remove the unused values in this file